### PR TITLE
Use o-typography for consistent typography styles

### LINKED ISSUE #176: Introduce o-typography
For spacing and typography. https://github.com/Financial-Times/o-overlay/blob/master/src/scss/overlay.scss#L36

### PR DESCRIPTION
**Pull request by rowanmanning**
This adds in o-typography. I haven't changed any of the spacing values,
as I might investigate using o-spacing for that.

Resolves #176

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-overlay bundle size difference from 2.7.11 to d1c937022c2720227a362318d759a84fcfe74be7
js: 0.51kb increase (0.15kb/gzip)
css, master: 0.27kb increase (0.11kb/gzip)
css, internal: 0.27kb increase (0.11kb/gzip)
An insignificant difference was also found for: css (whitelabel)

**Comment by origamiserviceuser** [comment]
o-overlay bundle size difference from 2.7.11 to 614941c1694ba12b1327f740c9ebf635f4de6073
js: 0.51kb increase (0.15kb/gzip)
css, master: 0.27kb increase (0.11kb/gzip)
css, internal: 0.27kb increase (0.11kb/gzip)
An insignificant difference was also found for: css (whitelabel)

File: bower.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "o-layers": "^2.1.0",
     "o-viewport": "^3.3.3",
     "o-visual-effects": "^2.0.0",
-    "o-brand": "^3.2.4"
+    "o-brand": "^3.2.4",
+    "o-typography": "^5.12.0"
```
Comment (JakeChampion) [inline bower.json]: Should we update the MIGRATION.md in this pull-request as well or in another pull-request?

Comment (rowanmanning) [reply bower.json]: Yes! I knew I'd forgotten something – we should mention this change I think

**Comment by origamiserviceuser** [comment]
o-overlay bundle size difference from 2.7.11 to 807031f44a9218b7c1909b637f784d1592000960
js: 0.51kb increase (0.16kb/gzip)
css, master: 0.27kb increase (0.11kb/gzip)
css, internal: 0.27kb increase (0.11kb/gzip)
An insignificant difference was also found for: css (whitelabel)

**Pull request merged**